### ISSUE TITLE
 Disabling SQN check in SysmoISIM-SJA2

### ISSUE THREAD

**Issue by MAlexVR** [herlesupreeth/sysmo-usim-tool#1]
Dear Herle,

I would like to test VoLTE with a  couple ofsamsung galaxy S1+ 5G. On step 23, I'm following your guide with SysmoISIM-SJA2 but I don't know how to disable SQN check with the current files. Could you explain me, please?

**Comment by herlesupreeth** [comment]
I dont think you need to disable the SQN check anymore for VoLTE to work. Just make sure to have the same SQN as in open5gs Web GUI written in FHoSS before you attempt to attach the phone to network
